### PR TITLE
feat: Initial Games Bet endpoint

### PR DESCRIPTION
**Pull request by tommy-trombone**
### Description

This endpoint is for making a bet. I created a new branch instead of handling merge conflicts between multiple endpoints created at the same time.

### How to test

Just endpoint scaffolding/skeleton at the moment

### Task

[CU-86cvbdgx6](https://app.clickup.com/t/86cvbdgx6)

### CODE REVIEW

**Comment by richardaspinall** [comment]
Task linked: [CU-86cvbdgx6 [BE] generate actions endpoints](https://app.clickup.com/t/86cvbdgx6)

File: backend/src/shared/api/metadata/games_bet.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    "authRequired": true,
+    "payload": {
+      "properties": [
+        {
+          "name": "pokerTableName",
+          "type": "string",
+          "required": true
+        },
+        {
+          "name": "betAmount",
```
Comment (richardaspinall) [inline backend/src/shared/api/metadata/games_bet.json]: could we just have this as `amount`?

**Review by richardaspinall — CHANGES_REQUESTED** [review]
Looks good. One minor thing!

File: backend/src/shared/api/metadata/games_bet.json (hunk 1 above)
Comment (tommy-trombone) [reply backend/src/shared/api/metadata/games_bet.json]: Yup!

**Review by richardaspinall — APPROVED** [review]

**Pull request merged**